nym-vpn-core/crates/nym-vpnd/src/cli.rs

```diff
@@ -48,8 +48,13 @@ pub(crate) struct Command {
     #[arg(long)]
     pub(crate) start: bool,
 
+    #[cfg(windows)]
     #[arg(long)]
     pub(crate) run_as_service: bool,
+
+    #[cfg(unix)]
+    #[arg(long)]
+    pub(crate) run_as_cli: bool,
```

Comment:
Maybe make this only for mac? Otherwise you'll need to edit all the systemd stuff and vpnx installers too